Jonathan-jonathankern/Homework-2/temp-converter/js/main.js

```diff
@@ -0,0 +1,111 @@
+*/
+
+// assign #fahrenheit_to_celsius variable
+let fahrenheitToCelsius = document.getElementById('fahrenheit_to_celsius');
+
+// assign #celsius_to_fahrenheit variable
+let celsiusToFahrenheit = document.getElementById('celsius_to_fahrenheit');
+
+// assign #celsius_to_fahrenheit variable
+let clearContents = document.getElementById('clear_contents');
+
+let showResult = document.getElementById('result');
+
+// add click event listener to fahrenheitToCelsius
+let fahrenheitToCelsiusEventHandler = fahrenheitToCelsius.addEventListener("click", function() {
+    // console.log("fahrenheit was clicked");
+
+    // store temperature from input field
+    let temperature = document.querySelector('#temperature').value;
+
+    fahrenheitToCelsiusConverter(temperature);
+});
+
+// add click event listener to celsiusToFahrenheit
+let celsiusToFahrenheitEventHandler = celsiusToFahrenheit.addEventListener("click", function() {
+    // console.log("celsius was clicked");
+
+    // store temperature from input field
+    let temperature = document.querySelector('#temperature').value;
+
+    celsiusToFahrenheitConverter(temperature);
+});
+
+// add click event listener to clearContents
+let clearContentsEventHandler = clearContents.addEventListener("click", function() {
+    // console.log("clearContents was clicked");
+    // store temperature from input field
+    let temperature = document.querySelector('#temperature').value = ' ';
+    // console.log(temperature);
+
+    reset(temperature);
+});
+
+// convert fahrenheit to celsius
+function fahrenheitToCelsiusConverter(temperature) {
+    // console.log(temperature + " " + "fahrenheit");
+
+    let convertedTemperature = (temperature - 32) / 1.8;
+    // console.log(convertedTemperature);
+
+    // round to whole number
+    let roundToWholeNumber = Math.round(convertedTemperature);
+    // console.log(temperature + " " + "converted from Fahrenheit to Celsius is" + " " + roundToWholeNumber);
+
+    showResult.innerHTML = '"' + temperature + '°' + " " + "converted from Fahrenheit to Celsius is" + " " + roundToWholeNumber + '°"';
+
+    reset(temperature);
+}
+
+// convert celsius to fahrenheit
+function celsiusToFahrenheitConverter(temperature) {
+    // console.log(celsiusTemperature + " " + "celsius");
+
+    let convertedTemperature = (1.8 * temperature) + 32;
+    // console.log(convertedTemperature);
+
+    // round to whole number
+    let roundToWholeNumber = Math.round(convertedTemperature);
+    // console.log(temperature + " " + "converted from Celsius to Fahrenheit is" + " " + roundToWholeNumber);
+
+    showResult.innerHTML = '"' + temperature + '°' + " " + "converted from Celsius to Fahrenheit is" + " " + roundToWholeNumber + '°"';
+
+    reset(temperature);
+}
+
+// reset form
+function reset(temperature) {
```

Comment:
It looks like you had to do some arm wrestling here because you were using a space to zero things out. If you use an empty string instead, the placeholder will be hidden/displayed automatically and you won't have to touch it with your JS.